monitoring/docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  prometheus:
+    image: prom/prometheus
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+  grafana:
+    image: grafana/grafana
+    ports:
+      - "3000:3000"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)